pandas/tests/arrays/floating/test_arithmetic.py

```diff
@@ -180,3 +180,43 @@ def test_cross_type_arithmetic():
     result = df.A + df.B
     expected = pd.Series([2, np.nan, np.nan], dtype="Float64")
     tm.assert_series_equal(result, expected)
+
```

Comment:
i think it's possible to be more more DRY and consolidate these tests into one

Reply:
consolidated into a single test `test_unary_float_operators`